### DISCUSSION TITLE
Would like to have prefer language selection (to change the results from default to other) like classic search

### DISCUSSION THREAD

**Discussion by GangadharMFord** [microsoft-search/pnp-modern-search#2658] [Ideas]
Hi Team,

Just would like to cross check whether we have an option / develop to choose a language for search results to change from default to other like at classic search.

![image](https://user-images.githubusercontent.com/111119555/208674161-79aaa6fd-504c-4bb3-94bc-cd61e27a7b33.png)

or is that could be possible with customizing the pnp search box?

**Comment by wobba** [comment]
This requires development and integration into the result web part. The logic today picks the UX language, or the language of a translated SharePoint page, if you are using this.
